tests/test_codebase/test_mmpose/test_mmpose_models.py

```diff
@@ -188,3 +188,33 @@ def test_scale_forward(backend_type: Backend):
         deploy_cfg=deploy_cfg,
         run_with_backend=False)
     torch_assert_close(rewrite_outputs, model_outputs)
+
+
+@pytest.mark.parametrize('backend_type', [Backend.ONNXRUNTIME])
+def test_yolox_pose_head(backend_type: Backend):
+    from mmdeploy.apis.utils import build_task_processor
+    from mmdeploy.utils import get_input_shape, load_config
+    check_backend(backend_type, True)
+    deploy_cfg, model_cfg = load_config(
+        'configs/mmpose/yolox-pose_onnxruntime_static.py',
+        '../mmpose/projects/yolox-pose/configs/yolox-pose_s_8xb32-300e_coco.py'
```

Comment:
put a new model config file into `tests/test_codebase/test_mmpose/data` and use it